src/Storage/StorageBucket.php

```diff
@@ -31,9 +32,19 @@ class StorageBucket
 	 */
 	protected array $headers = [];
 
-	public function __construct($url, $headers)
+	/**
+	 * StorageBucket constructor.
+	 *
+	 * @throws Exception
+	 */
+	public function __construct()
```

Comment:
The signature of this class should be:
```
public function __construct($reference_id, $api_key)
```

Then we should pass in those vars when instantiated. This makes it impossible to use this as a library without the DotEnv library. For testing, we can setup a .env.test that can contain those keys. And then pass them into the test as needed.